File: javascript/rna-transcription/rna-transcription.js
export const toRna = (dna_strand) => {
  let nucleotides = {'C': 'G', 'G': 'C', 'T': 'A', 'A': 'U'};
  let rna_strand = '';

  for (const nucleotide of dna_strand) {
    rna_strand += nucleotides[nucleotide];
  }

  return rna_strand;
};

File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    nucleotides = {'C': 'G', 'G': 'C', 'T': 'A', 'A': 'U'}
    rna_strand = ''

    for nucleotide in dna_strand:
        rna_strand += nucleotides[nucleotide]

    return rna_strand
